tests/system/providers/google/cloud/dataproc/example_dataproc_gke.py

```diff
@@ -55,7 +55,11 @@
     "workload_identity_config": {
         "workload_pool": WORKLOAD_POOL,
     },
-    "initial_node_count": 1,
+    "node_pools": [
```

Comment:
Hi!
With this change the error is like this:
`  File "/opt/airflow/airflow/providers/google/cloud/hooks/kubernetes_engine.py", line 297, in create_cluster
    operation = self.get_cluster_manager_client().create_cluster(
  File "/usr/local/lib/python3.8/site-packages/google/cloud/container_v1/services/cluster_manager/client.py", line 1079, in create_cluster
    response = rpc(
  File "/usr/local/lib/python3.8/site-packages/google/api_core/gapic_v1/method.py", line 131, in __call__
    return wrapped_func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/google/api_core/grpc_helpers.py", line 78, in error_remapped_callable
    raise exceptions.from_grpc_error(exc) from exc
google.api_core.exceptions.InvalidArgument: 400 Node_pool.name must be specified. [stack_entries: "cloud/kubernetes/engine/common/error_desc.go:456 +0x26 google3/cloud/kubernetes/engine/common/errdesc.(*GKEErrorDescriptor).createErr(0xc001a3bba0, {0x559eb23cad88, 0x559ebbdbb2e0})"
stack_entries: "cloud/kubernetes/engine/common/error_desc.go:314 +0x4c google3/cloud/kubernetes/engine/common/errdesc.(*GKEErrorDescriptor).WithMsgCtx(0x0?, {0x559eb23cad88?, 0x559ebbdbb2e0?}, {0xc18acce1e0, 0x20}, {0x0, 0x0, 0x0})"
stack_entries: "cloud/kubernetes/engine/common/error_desc.go:306 google3/cloud/kubernetes/engine/common/errdesc.(*GKEErrorDescriptor).WithMsg(...)"
stack_entries: "cloud/kubernetes/server/validation/input_validation.go:137 +0x265 google3/cloud/kubernetes/server/validation/validate.ClusterName({0x0?, 0x9?}, {0x559ea3d77eed?, 0xc005101008?})"
stack_entries: "cloud/kubernetes/server/v1alpha1/input_validation.go:1360 +0x56 google3/cloud/kubernetes/server/v1alpha1/validate.(*Validator).V1alpha1NodePool(0xc054d8cc58, {0x559eb23caca8, 0xc1c66f21b0}, 0xc03320adc0)"
stack_entries: "cloud/kubernetes/server/v1alpha1/input_validation.go:188 +0x439 google3/cloud/kubernetes/server/v1alpha1/validate.(*Validator).V1alpha1Cluster(0xc054d8cc58, {0x559eb23caca8, 0xc1c66f21b0}, 0xc059e3aa08, {0xc157ae3bad, 0xb})"
stack_entries: "cloud/kubernetes/server/v1alpha1/input_validation.go:651 +0xfc google3/cloud/kubernetes/server/v1alpha1/validate.(*Validator).V1alpha1CreateClusterRequest(0xc054d8cc58, {0x559eb23caca8, 0xc1c66f21b0}, 0xc0c4e3c150)"
stack_entries: "cloud/kubernetes/server/v1alpha1/server.go:142 +0x77 google3/cloud/kubernetes/server/v1alpha1/server.(*ClusterServer).createCluster(0xc061b91100, {0x559eb23caca8, 0xc1c66f21b0}, 0xc0c4e3c150, 0xc00e6f3b00, 0xc4cf6f8b70)"
stack_entries: "cloud/kubernetes/server/v1alpha1/server.go:121 +0x331 google3/cloud/kubernetes/server/v1alpha1/server.(*ClusterServer).CreateCluster(0xc061b91100, {0x559eb23caca8, 0xc1c66f21b0}, 0xc0c4e3c150, 0xc00e6f3b00)"
stack_entries: "cloud/kubernetes/engine/server/api/v1/server.go:38 +0x105 google3/cloud/kubernetes/engine/server/api/v1/server.(*ClusterServer).CreateCluster(0xc054d8cc68, {0x559eb23caca8, 0xc1c66f21b0}, 0xc3b42abb90, 0xc00e6f3a00)"
detail: "INVALID_ARGUMENT: node_pool.name must be specified"
, request_id: "0x4c38a7fc81c44a9b"
]
[2024-05-28, 17:14:15 UTC] {local_task_job_runner.py:240} INFO - Task exited with return code 1`